salt/annotations/config/home-deployuser-.env

```diff
@@ -0,0 +1,2 @@
+COMPOSE_PROJECT_NAME=annotations
+IMAGE_TAG={{ salt['elife.rev']('latest') }}
```

Comment:
This allows for pinning a particular commit, assuming the image deriving from that commit has been built and pushed by Alfred.

Reply:
See list at https://hub.docker.com/r/elifesciences/annotations_fpm/tags/